java-frontend/src/main/java/org/sonar/java/se/checks/FlowComputation.java

```diff
@@ -128,9 +128,9 @@ private static String learnedConstraintMessage(Constraint lc, ExplodedGraph.Node
       return String.format("'%s' returns %s", name, lc.valueAsString());
     }
     if (nodeTree.is(Tree.Kind.NEW_CLASS)) {
-      return String.format("'new %s' returns %s", name, lc.valueAsString());
+      return String.format("Constructor call creates '%s'", lc.valueAsString());
     }
-    return name == null ? lc.valueAsString() : String.format("assume '%s' is %s", name, lc.valueAsString());
+    return name == null ? lc.valueAsString() : String.format("Implies '%s' is %s", name, lc.valueAsString());
```

Comment:
It was proposed by Jean-Christophe to use capital letters https://jira.sonarsource.com/browse/SONARJAVA-2003?focusedCommentId=111014&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-111014